test/cli_test.py

```diff
@@ -373,7 +408,47 @@ def test_no_move_on_extension_change(self):
         self.assertFileTag(converted_path, b'ISMP3')
 
 
+class ExternalConvertWorkerTest(TestHelper):
+    """Test alternatives with non-empty ``format `` option, i.e. transcoding
+    some of the files. In contrast to the previous test, these test do not
+    work the parallelizing Worker.
```

Comment:
I think you mistyped. These tests _do_ use the production worker

Reply:
Indeed I did...